pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
Ah nevermind, that's a different set of changes from https://github.com/docker/docker/compare/v1.10.2...hallyn:v1.10.0.serge.2#diff-4166b9ad558bc9c8d0ff7b01b69c128aR34 causing that one -- carry on! :+1: